### PR TITLE
Adds a reasonable sound for msay messages.

### PR DESCRIPTION
**Pull request by Probe1**
https://notificationsounds.com/message-tones/stairs-567

https://github.com/vgstation-coders/vgstation13/pull/24484

:cl:
 * rscadd: new sound for msay messages

### CODE REVIEW

**Comment by boy2mantwicethefam** [comment]
bring back waagh

**Comment by MisterStaub** [comment]
as long as its toggleable its gucci

**Comment by d3athrow** [comment]
Make it this and add a preference
https://www.myinstants.com/instant/aim-incoming-message-33364/

**Comment by GeneralVeers25** [comment]
Fuck no. Why does this need a sound? At least give us the ability to toggle it (and other sound effects) off.

**Comment by Zth--** [comment]
I think it should be even more subtle

**Comment by SysadminJeroen** [comment]
I want the RCT notification sound.

**Comment by Probe1** [comment]
If you have a better sound by all means post it here.

**Comment by bathosbathtime** [comment]
Redundant given you can already use adminhelp for the same effect but otherwise fine if a toggle was included.

File: code/modules/admin/verbs/adminsay.dm [hunk 1]
```diff
@@ -47,3 +47,4 @@
 	for(var/client/C in admins)
 		if((R_ADMIN|R_MOD) & C.holder.rights)
 			C.output_to_special_tab("<span class='prefix'>MOD: </span><span class='[color]'><EM>[key_name(src,1)]</EM> (<A HREF='?src=\ref[C.holder];adminplayerobservejump=\ref[mob]'>JMP</A>): <span class='modooc'>[msg]</span></span>")
+			C << 'sound/effects/msaysound.ogg'
```
Comment (MadmanMartian) [inline code/modules/admin/verbs/adminsay.dm:50]: preference option or riot

**Comment by Kammerjunk** [comment]
oh wow there really isn't a proc for playing a sound to a player if the sound has no source

File: code/modules/admin/verbs/adminsay.dm (hunk 1 above)
Comment (Probe1) [reply code/modules/admin/verbs/adminsay.dm:50]: Help me add one please?

Comment (ShiftyRail) [reply code/modules/admin/verbs/adminsay.dm:50]: Trying to make a PR to automatise making a new pref right now

**Comment by ShiftyRail** [comment]
See #24873 for documentation/help on how to make it a preference.

**Comment by Kurfursten** [comment]
Check out that new script

**Comment by Goldsteel** [comment]
I agree that the sound should have a toggle. I'm 50/50 on the default status, though.

**Pull request closed without merging**